### PR TITLE
Add steps to get credentials for C* access

### PR DESCRIPTION
**Pull request by johnsmartco**
plus notes about matching S3 keys, bucket name, and region

**What this PR does**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [x] Changes manually tested
- [ ] Chart versions updated (if necessary)
- [ ] Automated Tests added/updated
- [x] Documentation added/updated
- [x] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: docs/content/en/docs/topics/restore-a-backup/_index.md
Comment (bradfordcp) [inline docs/content/en/docs/topics/restore-a-backup/_index.md:210]: Consider adding a comment here that this value is for your demo cluster. The value will be different on their installation.

```suggestion
# Note this value is for _this_ cluster. Your cluster will have a different value.
7kZV7YUFSWUTksJ9nfZoWzuTL0qGSgjp54kEVAlgrbMTW_E3RXcTsg
```

**Comment by bradfordcp** [comment]
These changes look great @johnsmartco.

File: docs/content/en/docs/topics/restore-a-backup/_index.md
Comment (johnsmartco) [reply docs/content/en/docs/topics/restore-a-backup/_index.md:210]: Inc'd edit.

File: docs/content/en/docs/topics/restore-a-backup/_index.md
Comment (jsanda) [inline docs/content/en/docs/topics/restore-a-backup/_index.md:35]: https://github.com/k8ssandra/k8ssandra/pull/317 has changes to allow using different regions for the S3 bucket. I hope to get it merged tomorrow. We will want to add docs around the new properties that are introduced in the PR.

File: docs/content/en/docs/topics/restore-a-backup/_index.md
Comment (jsanda) [inline docs/content/en/docs/topics/restore-a-backup/_index.md:199]: A couple things here. First, I would change `username secret` to `superuser secret`. Secondly it defaults to `<cluster-name>-superuser`. The cluster name defaults to the release name.  Let's say you install as follows:

```
$ helm install demo k8ssandra/k8ssandra
```
The release name is `demo` which means the cluster name will also default to `demo`. The superuser secret will then be named `demo-superuser`.

**Review by jsanda — CHANGES_REQUESTED** [review]

File: docs/content/en/docs/topics/restore-a-backup/_index.md
Comment (johnsmartco) [reply docs/content/en/docs/topics/restore-a-backup/_index.md:35]: Ok sounds good - thanks John. We might as well wait for that code merge before proceeding with this doc merge.

File: docs/content/en/docs/topics/restore-a-backup/_index.md
Comment (johnsmartco) [reply docs/content/en/docs/topics/restore-a-backup/_index.md]: Incorporated comments from John Sanda. Please check that with cluster-name of demo, all corresponding changes are correct in examples.

File: docs/content/en/docs/topics/restore-a-backup/_index.md
Comment (johnsmartco) [reply docs/content/en/docs/topics/restore-a-backup/_index.md]: Removed note and text about matching region for S3

**Review by johnwfrancis — APPROVED** [review]

**Comment by johnsmartco** [comment]
Guys, I think I've incorporated all the requested edits.  Ok to proceed?  If you agree, +1 approve. Thanks.  @jsanda @jdonenine @bradfordcp

File: docs/content/en/docs/topics/restore-a-backup/_index.md
Comment (jsanda) [inline docs/content/en/docs/topics/restore-a-backup/_index.md:205]: We should have a section in the docs that describe the steps necessary for accessing C* credentials and accessing cqlsh (not sure if that exists yet). Should we just link to that section here?

File: docs/content/en/docs/topics/restore-a-backup/_index.md
Comment (jsanda) [inline docs/content/en/docs/topics/restore-a-backup/_index.md:256]: We should be referencing the chart repo in the docs. The command below should be:

```
helm install test k8ssandra/backup --set name=test,cassandraDatacenter.name=dc1
```

File: docs/content/en/docs/topics/restore-a-backup/_index.md
Comment (jsanda) [inline docs/content/en/docs/topics/restore-a-backup/_index.md:325]: I think we should change this from `Repair Web Interface` to `Reaper Web Interface` since it is specific to Reaper.

File: docs/content/en/docs/topics/restore-a-backup/_index.md
Comment (johnsmartco) [reply docs/content/en/docs/topics/restore-a-backup/_index.md:204]: We'll have a separate topic or a section with the Quick Start that describes accessing C* credentials and cqlsh, yes. But users get lost if we xref them to another topic in the middle of a step-by-step procedure. So I'm inclined to explicitly show the credentials examples here too, even though it duplicates content.

File: docs/content/en/docs/topics/restore-a-backup/_index.md
Comment (johnsmartco) [reply docs/content/en/docs/topics/restore-a-backup/_index.md:255]: Inc'd correct command - thanks John

File: docs/content/en/docs/topics/restore-a-backup/_index.md
Comment (johnsmartco) [reply docs/content/en/docs/topics/restore-a-backup/_index.md]: Done - thanks

**Review by jsanda — APPROVED** [review]

**Pull request merged**
